corehq/apps/dhis2/dbaccessors.py

```diff
@@ -12,3 +12,16 @@ def get_dhis2_connection(domain_name):
         reduce=False,
     ).first()
     return Dhis2Connection.wrap(result['doc']) if result else None
+
+
+@quickcache(['domain_name'])
```

Comment:
not sure how often the data would change but if it's not often you should increase the timeout here (or just clear the cache when it does change)